frontend/assets/sass/main.scss

```diff
@@ -9,22 +9,26 @@
 @import "abstracts/placeholders";
 @import "abstracts/vars.color";
 @import "abstracts/vars.type";
-@import "abstracts/vars.uikit";
 
 
-// 2. Vendor & plugin styles.
-@import "uikit/src/scss/variables-theme.scss";
-@import "uikit/src/scss/mixins-theme.scss";
-@import "uikit/src/scss/uikit-theme.scss";
+// 2. UIKit Core & Theme.
+@import "theme/variables"; // needs to come before variables-theme
+@import "uikit/src/scss/variables-theme";
```

Comment:
Assuming we _do_ want to include the UIKit theme vars & mixins and build up from there, shouldn't our vars come _after_ the UIKit ones?